_topic_map.yml

```diff
@@ -2254,6 +2254,8 @@ Topics:
       File: virt-moving-local-vm-disk-to-different-node
     - Name: Expanding virtual storage by adding blank disk images
       File: virt-expanding-virtual-storage-with-blank-disk-images
+    - Name: Cloning a DataVolume using smart-cloning
```

Comment:
There is inconsistency between using "smart cloning" and "smart-cloning". Determine which, and then use throughout the PR.

Reply:
Smart-cloning (with hyphen) should now be present in all occurrences.